test/object-store/collection_fixtures.hpp

```diff
@@ -601,7 +601,8 @@ struct ListOfMixedLinks : public LinkedCollectionBase {
         Lst<Mixed> list = obj.get_list<Mixed>(col);
         size_t num_unresolved = 0;
         for (auto value : list) {
-            if (value.is_unresolved_link()) {
+            // mixed link can never be invalid only null
+            if (value.is_null()) {
```

Comment:
Changing these testing structures is hiding the issue. You will likely have to update any failing tests to reflect the changed behaviour of the containers.

Reply:
The test is failing and rightfully so, since we are now treating all the unresolved links as they were nulls, so I believe this fix is correct.

Reply:
This is the only test that is failing, and rightfully because we're now treating unresolved links as nulls. I believe this fix is correct.

Reply:
I fixed this, checking if the test is running against `Set<Mixed>/List<Mixed>`